masterlist.yaml

```
@@ -10115,10 +10177,86 @@ plugins:
       - crc: 0x397B1E26
         util: 'SSEEdit v4.0.3'
 
+  - name: 'BlinkSpell.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/21606/' ]
+    msg: [ *patch3rdParty_SRPC ]
+
+  - name: 'BloodSpellsPack.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/15652/' ]
+    msg: [ *patch3rdParty_SRPC ]
+
+  - name: 'Blue Fire Magic - Fire.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/452/' ]
+    msg: [ *patch3rdParty_SRPC ]
+
+  - name: 'Blue Fire Magic - Frost.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/452/' ]
+    msg: [ *patch3rdParty_SRPC ]
```

These can be combined with a regex.

```suggestion
  - name: 'Blue Fire Magic - (Fire|Frost)\.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/452/' ]
    msg: [ *patch3rdParty_SRPC ]
```